test/XRP/fakes/fake-xrp-transaction-type-protobufs.ts

```diff
@@ -25,6 +31,10 @@ const testSetFlag = 4
 const testTransferRate = 1234567890
 const testTickSize = 7
 
+// AccountDelete values
+const testDestination = 'rPEPPER7kfTD9w2To4CQk6UCfuHM9c6GDY'
```

Comment:
super nit: seems like it's common in this file to put a space between the comment describing the values and the values themselves?